vsim/Makefrag-verilog

```diff
@@ -9,14 +9,10 @@ firrtl: $(generated_dir)/$(MODEL).$(CONFIG).fir
 
 .PHONY: firrtl
 
-$(generated_dir)/%.$(CONFIG).fir $(generated_dir)/%.$(CONFIG).d $(generated_dir)/%.prm: $(chisel_srcs) $(bootrom_img)
+$(generated_dir)/%.$(CONFIG).v $(generated_dir)/%.$(CONFIG).TestHarness.v $(generated_dir)/%.$(CONFIG).conf $(generated_dir)/%.$(CONFIG).fir $(generated_dir)/%.$(CONFIG).d $(generated_dir)/%.$(CONFIG).prm: $(chisel_srcs) $(bootrom_img)
```

Comment:
Nice you also fixed the %.prm bug that has recently been causing a double invocation bug.